templates/default/event/eventedit.html

```diff
@@ -166,11 +166,27 @@
 			<B>{trans("Users:")}</B>
 		</TD>
 		<TD width="98%" class="nobr">
-			<SELECT size="{if $max_userlist_size > 0}{$max_userlist_size}{else}{size of=$userlist}{/if}" name="event[userlist][]" {tip text="Select user(s) (optional)"} multiple>
-				{foreach $userlist as $user}
-				<OPTION value="{$user.id}"{if in_array($user.id, $event.userlist)} selected{/if}>{$user.name}</OPTION>
-				{/foreach}
-			</SELECT>
+                    <div style="float: left;">
+                        <select style="width: 201px;" size="10" name="event[userlist][]" {tip text="Select user(s) (optional)"} multiple>
+                            {foreach from=$userlist item=user}
+                                <option group="{$user.usergroupid}"{if $smarty.cookies.usergroup && $smarty.cookies.usergroup neq $user.usergroupid} style="display: none;"{/if} value="{$user.id}" {if in_array($user.id, $event.userlist)} selected{/if} {if $event.userlist && in_array($user.usergroupid, $event.userlist)} SELECTED{/if}>{$user.name}</option>
+                            {/foreach}
+                        </select>
+                    </div>
+                    <div style="float: left; padding-top: 30px;">
+                        <label for="usergroup_0">
+                            <input type="radio" name="event[usergroup]" id="usergroup_0" onChange="changeUserGroup(0)"{if !$smarty.cookies.usergroup} checked="checked"{/if}>&nbsp;
+                            {trans("All users")}
+                        </label>
+                        <br />
+                        {foreach from=$usergroup item=group}
+                            <label for="usergroup_{$group.id}">
+                                <input type="radio" name="event[usergroup]" id="usergroup_{$group.id}" onClick="changeUserGroup({$group.id})"{if $smarty.cookies.usergroup eq $group.id} checked="checked"{/if}>&nbsp;
+                                {$group.name}
+                            </label>
+                            <br />
```

Comment:
a co ma być ?? 
dałęm to po to żeby każdy radiobutton z opisem był w nowej lini
wygląda to tak:
![image](https://cloud.githubusercontent.com/assets/15612188/13779284/8633d3b8-eaba-11e5-972f-9d50f77f6ac3.png)

bez br tak:
![image](https://cloud.githubusercontent.com/assets/15612188/13779311/a1701452-eaba-11e5-8d56-192ec7fa555e.png)